frontend/src/components/widgets/Form/FormSubmitContent.tsx

```diff
@@ -0,0 +1,14 @@
+import React, { ReactElement } from "react"
+import { FormSubmitButton, Props } from "./FormSubmitButton"
+import { StyledFormSubmitContent } from "./styled-components"
+
+export default function FormSubmitContent(props: Props): ReactElement {
```

Comment:
Try exporting it without `default`. In that case the bottom export would just become `export { FormSubmitContent } from "./FormSubmitContent"`